test/security/testrunners/filepermissiontestrunner.go

```diff
@@ -0,0 +1,108 @@
+package testrunners
```

Comment:
Maybe move to the internal folder and create a "security folder" with these security check
https://github.com/aws/amazon-cloudwatch-agent-test/tree/main/filesystem

Reply:
created an acceptance folder as people suggested in the slack check